eng/pipelines/templates/stages/archetype-sdk-tests.yml

```diff
@@ -143,7 +143,7 @@ stages:
               Cloud: ${{ cloud.key }}
 
 - ${{ if contains(variables['Build.DefinitionName'], 'tests-weekly')  }}:
-  - stage: Analysis
+  - stage: ${{ parameters.JobName }}_Analysis
```

Comment:
For KV, we have stages that are named as `{cloud}_{package-name}` -- if the same package is being tested multiple times for each cloud, could this still have collisions? Or is the analysis stage just run once for each package, rather than each cloud/package combination?

Reply:
> For my information what cases do we have that includes this template twice?

If you check `sdk/keyvault/tests.yml` you will find that we call into `archetype-sdk-tests` multiple times.